ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -359,6 +359,8 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.mpls.ldp.igp_sync is arista.avd.defined(true) %}
    mpls ldp igp sync
 {%     endif %}
+{%     set interface_ip_nat = port_channel_interface.ip_nat | arista.avd.default({}) %}
```

Comment:
please also add the `if` block around this one. same for vlan_interfaces. Thanks

Reply:
Sorry for missing the two sections in last fix commit.